includes/functions.php

```diff
@@ -733,10 +750,10 @@ function object_to_uri( $data ) {
 			$data = object_to_uri( $data['url'] );
 			break;
 		case 'Link':
-			$data = $data['href'];
+			$data = $data['href'] ?? '';
 			break;
 		default:
-			$data = $data['id'];
+			$data = $data['id'] ?? $data['url'] ?? '';
```

Comment:
Apologies for being a stickler here. These still would require adding `empty()` checks before being able to use the return value. Is this still about making the `array_filter()` call work in `extract_recipients_from_activity_property()`?

Reply:
Currently we can be reasonably sure that the output of `object_to_uri()` is a URI that we can use right away. I'd consider PHP warnings/fatals a good thing, actually, as a signal that there's something missing in this function or in object validation before this function is called.

Up until now, we don't know of any instance where we need a fallback to an empty string. Let's not introduce one just in case, but rather make fixes as we encounter problems, so `object_to_uri()`stays as reliable as it has been.